src/components/Testimonials.js

```diff
@@ -9,33 +9,61 @@ const Testimonials = props => (
     <ul className="features">
```

Comment:
Changes in this file include: 
- switching the spots of Michelle and Yena's testimonials
- updating text to Yena's testimonial
- adding classname `.two` to the two testimonials on the bottom

everything else are formatting changes via Prettier